src/content/variables.json

```diff
@@ -3,7 +3,5 @@
   "kubernetesMaxVersion": "1.30",
   "cliVersion": "3.2.1",
   "chartVersion": "1.27.0",
-  "nginxVersion": "v1.11.2",
-  "reloaderVersion": "v1.0.119",
-  "redisVersion": "7.4.0-debian-12-r1"
+  "syncthingVersion": "1.28.0"
```

Comment:
I fear that this variable is going to be very hard to remember to update in each version. Can we run a script to set this one? For example:
``` bash
$ docker run -it okteto/okteto:3.2.1 /usr/bin-image/bin/syncthing --version \
  | grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+' \
  | sed 's/v//'
1.28.0 
```
If we could do this, it will be much easier to maintain